zookeeper-contrib/zookeeper-contrib-zooinspector/pom.xml

```diff
@@ -70,8 +70,8 @@
       </exclusions>
     </dependency>
     <dependency>
-      <groupId>org.junit.vintage</groupId>
-      <artifactId>junit-vintage-engine</artifactId>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-engine</artifactId>
       <scope>test</scope>
     </dependency>
```

Comment:
There are no tests at all in `zooinspector`.  Should we remove this "dependency"?